src/main/resources/feedbackQuestionMcqSubmissionFormOptionFragment.html

```diff
@@ -1,9 +1,11 @@
 <tr>
     <td>
-        <label><input type="radio"
-            name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${qnIdx}-${responseIdx}"
-            ${disabled}
-            ${checked}
-            value="${mcqChoiceValue}"> ${mcqChoiceValue}</label>
+        <div class="radio">
```

Comment:
These are to ensure long label values wrap correctly to the next line, not under the checkbox/radio button but indented to match the first line.